### PR TITLE
Fix a small typo and some grammar mistakes

### PR DESCRIPTION
**Pull request by xspynks**

### CODE REVIEW

**Comment by jsa34** [comment]
Looks good to me

**Review by NotMyFault — APPROVED** [review]
Looks fine, but please don't delete the PR template.

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
